content/get-started/exploring-projects-on-github/following-people.md

```diff
@@ -12,11 +12,35 @@ versions:
 topics:
   - Profile
 ---
-When you follow people, you'll see their public activity in the activity view of your news feed.{% ifversion fpt %} If someone you follow stars a public repository, {% data variables.product.product_name %} may recommend the repository to you in the discovery view of your news feed.{% endif %} To follow someone, visit their profile page and click **Follow** under their profile image.
 
-![Follow user button](/assets/images/help/profile/follow-user-button.png)
+## About followers on {% data variables.product.product_name %}
+
+When you follow people, you'll see their public activity in the activity view of your news feed.{% ifversion fpt %} If someone you follow stars a public repository, {% data variables.product.product_name %} may recommend the repository to you in the discovery view of your news feed{% endif %}.
```

Comment:
Sorry, just realized that we still have `..` in the GHES and GHAE version of the docs...

```suggestion
When you follow people, you'll see their public activity in the activity view of your news feed.{% ifversion fpt %} If someone you follow stars a public repository, {% data variables.product.product_name %} may recommend the repository to you in the discovery view of your news feed.{% endif %}
```